### PR TITLE
[CP-1817] [Sequential update] update check status disappears

### PR DESCRIPTION
**Pull request by OskarMichalkiewicz**
Jira: [CP-1817]

Check for update flow adjusted. To check the feature:
 * wait for initial "silent" check for os update,
 * turn off the internet connection,
 * check for update without internet connection,
 * turn the internet back on,
 * check for update again.

<details>
<summary><b>Screenshots</b></summary>

obraz

</details>

[CP-1817]: https://appnroll.atlassian.net/browse/CP-1817?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by patryk-sierzega — APPROVED** [review]
LGTM ;)

File: packages/app/src/overview/components/overview-screens/pure-overview/pure-overview.component.tsx [hunk 1]
```diff
@@ -92,6 +93,7 @@ export const PureOverview: FunctionComponent<PureOverviewProps> = ({
     forceUpdateNeeded,
     osVersion,
   })
+  console.log(checkForUpdateLocalState)
```
Comment (lkowalczyk87) [inline packages/app/src/overview/components/overview-screens/pure-overview/pure-overview.component.tsx:96]: :)

File: packages/app/src/__deprecated__/main/main.ts [hunk 2]
```diff
@@ -246,8 +246,11 @@ app.on("window-all-closed", () => {
 })
 
 app.on("activate", () => {
-  if (win === null) {
-    void createWindow()
+  const allWindows = BrowserWindow.getAllWindows()
+  if (allWindows.length) {
+    allWindows[0].focus()
+  } else {
+    createWindow()
```
Comment (patryk-sierzega) [inline packages/app/src/__deprecated__/main/main.ts:253]: please, add this void :)

File: packages/app/src/__deprecated__/main/main.ts [hunk 3]
```diff
@@ -515,3 +518,11 @@ ipcMain.answerRenderer(
 ipcMain.answerRenderer(OutlookAuthActions.CloseWindow, () => {
   outlookAuthWindow?.close()
 })
+
+app.on("second-instance", () => {
+  if (win) {
+    // Focus on the main window if the user tried to open another
+    if (win.isMinimized()) win.restore()
```
Comment (patryk-sierzega) [inline packages/app/src/__deprecated__/main/main.ts:525]: please, add brackets to if :)

**Review by patryk-sierzega — CHANGES_REQUESTED** [review]

File: packages/app/src/__deprecated__/main/main.ts (hunk 3 above)
Comment (OskarMichalkiewicz) [reply packages/app/src/__deprecated__/main/main.ts:525]: those changes wasn't supposed to be there. Going to delete them i a sec : O

**Review by patryk-sierzega — APPROVED** [review]
LGTM :)

**Review by lkowalczyk87 — APPROVED** [review]

**Pull request merged**